public/js/display.js

```diff
@@ -624,6 +648,26 @@ function attachMessage(
 ) {
  const content = elem()
 
+ // --- Add expiration warning for negative scores ---
+ if (message.score < 0) {
```

Comment:
The score of the message will not be negative, the velocity will be negative

The score is positive meaning it still has time to expire

If the score is 100 and the velocity is -10 per hour that means it will expire in 100/10 = 10 hours